bindings/python/pymongoarrow/context.py

```diff
@@ -23,7 +23,8 @@
     _BsonArrowTypes.int64: Int64Builder,
     _BsonArrowTypes.double: DoubleBuilder,
     _BsonArrowTypes.datetime: DatetimeBuilder,
-    _BsonArrowTypes.objectid: ObjectIdBuilder
+    _BsonArrowTypes.objectid: ObjectIdBuilder,
+    _BsonArrowTypes.string: StringBuilder
```

Comment:
Suggest adding a trailing comma to make future diffs nicer :)

Reply:
Done